plugins/cost-insights/src/types/Alert.ts

```diff
@@ -34,8 +34,8 @@ import { Maybe } from './Maybe';
  */
 
 export type Alert = {
-  title: string;
-  subtitle: string;
+  title: string | JSX.Element;
+  subtitle: string | JSX.Element;
```

Comment:
There is a slight gotcha here that shouldn't be blocking but I'll add here for transparency/visibility. 

Titles and subtitles are rendered in slightly different contexts on the page. In the Action Items section, they are passed to the `title` and `subheader` props of `CardHeader` and rendered as children of  `Typography` components in `AlertInsightsSectionHeader`. Each alert insights section header renders the title within a `h4` element.

Only [phrasing content](https://html.spec.whatwg.org/multipage/dom.html#phrasing-content-2) is allowed as a child of a header element - ideally just a plain text node. So a user can return a `span` element but not another `h4`, `div`, etc without violating HTML semantics.